reference/orders.v2.oas2.yml

```diff
@@ -1093,8 +1179,196 @@ paths:
+                        cost_tax_class_id: 2
+                        base_handling_cost: 4.1
+                        handling_cost_ex_tax: 4.1
+                        handling_cost_inc_tax: 5.3
+                        handling_cost_tax: 1.2
+                        handling_cost_tax_class_id: 2
+                        shipping_zone_id: 1
+                        shipping_zone_name: United States
+                        form_fields:
+                          - name: License Id
+                            value: 123BAF
+                        line_items:
+                          - url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/129/products/12'
+                            resource: /orders/129/products/12
+                        shipping_quotes:
+                          url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/163/consignments/shipping/64/shipping_quotes'
+                          resource: /orders/163/consignments/shipping/64/shipping_quotes
+                    downloads: [ ]
+                    email:
+                      gift_certificates: [ ]
+                Digital consignment:
+                  value:
+                    shipping: [ ]
+                    downloads:
+                      - recipient_email: recipient@email.com
+                        line_items:
+                          url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/129/products/12'
+                          resource: /orders/129/products/12
+                    email:
+                      gift_certificates: [ ]
+                Gift certificate consignment:
+                  value:
+                    shipping: [ ]
+                    downloads: [ ]
+                    email:
+                      gift_certificates:
+                        - recipient_email: recipient@email.com
+                          line_items:
+                            url: 'https://api.bigcommerce.com/stores/{store_hash}/v2/orders/129/products/12'
+                            resource: /orders/129/products/12
+        '404':
+          description: Not Found
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    status:
+                      type: integer
+                    message:
+                      type: string
+              examples:
+                response:
+                  value:
+                    - status: 404
+                      message: The requested resource was not found.
+      operationId: get-orders-orderId-consignments
+      description: 'List all of the order consignments. '
+      parameters:
+        - $ref: '#/components/parameters/store_hash'
+        - $ref: '#/components/parameters/order_id_path'
+  '/stores/{store_hash}/v2/orders/{order_id}/consignments/shipping/{shipping_consignment_id}/shipping_quotes':
```

Comment:
@bc-tgomez Are you able to successfully get a response to this request when you run it?